EveripediaNetwork/ep-ui
# fixed ui hydration error

# fixed ui hydration error

hydration error pops up when running this app on the local server, encountered error while trying to create a pull request

## How should this be tested?

1. start the app on the local server
2. refresh the page a couple times to ensure the error is properly tested


## Notes or observations

discovered .husky code differs across different machine
